nvmeadm/src/nvmf_discovery.rs

```diff
@@ -405,6 +405,46 @@ impl DiscoveryLogEntry {
         Ok(connect_args)
     }
 }
+
+///
+/// This method connects to a specific NVMf device available over tcp,
+/// identified by its ip address, port and nqn.
+///
+/// When we are already connected to the same host we will get back
+/// connection in progress. These errors are propagated back to use as
```

Comment:
Typo: "back to us" instead of "back to use"

Reply:
https://github.com/openebs/Mayastor/pull/302/commits/103a309bb8f0d25e4b4cf62544a0d24d400cd998